src/styles/card.scss

```diff
@@ -0,0 +1,73 @@
+.card {
+  width: 198px; // including border
+  height: 406px;
+  border: 1px solid $card-border-color;
```

Comment:
It's great to see that you are using variables for colors. This is a good practice for maintainability and theming. However, make sure that the variable `$card-border-color` is defined in your `variables.scss` or another appropriate place.